scripts/serveDocs.sh

```diff
@@ -7,4 +7,9 @@ echo "Changed directory to $(pwd)"
 
 npm run generate-cli-doc
 
-docker run --rm -it -p 8000:8000 -v $(pwd):/docs squidfunk/mkdocs-material:6.0.2
+# Build image if not existent
+if [[ "$(docker images -q squidfunk/mkdocs-material:8.5.9 2> /dev/null)" == "" ]]; then
```

Comment:
`squidfunk/mkdocs-material:8.5.9` is used several times, so might be possible to be stored in a variable so declaration only happens once